index.html

```diff
@@ -4,10 +4,55 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <title>Portfolio website </title>
 </head>
 <body>
-    <h1>My portfolio website</h1>
+    <main class="Portfolio-container">
+        <header>
+            <nav>
+               <a href="#" class="logo">RIDA</a> 
+             <a href="#" class="menu-container"><img src="Images/icon-menu.png" alt="menu-icon"></a>   
+            </nav>
+        </header>
+            <section class="headline-section">
+                <div class="primary-text"> 
+                    <h1>I’m RIDA <br> Glad to see <br> you!</h1>
+                    <p>
+                        I’m a software developer! I can help you build a product
+                         , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    </p>
+                    <hx class="supportive-text">
```

Comment:
Great work implementing all suggestions by the previous reviewer. However, you have one thing to change. When he mentioned `hx` he was referring to any of the six header elements in their hierarchy. Essentially replace that "x" with a number between 1 - 6. You will not use h1 any longer because you have already on your page. You could use h2 or h3.